packages/design-system/src/components/Dropdown/Primitive/DropdownEntry.module.scss

```
@@ -0,0 +1,47 @@
+@use '~@talend/design-tokens/lib/tokens';
+
+$iconAlignmentOffsetM: -0.3rem;
+
+.dropdownEntry {
+	background: tokens.$coral-color-accent-background-weak;
+	display: block;
+	width: 100%;
+	min-width: 0;
+	color: tokens.$coral-color-neutral-text;
+	font: tokens.$coral-paragraph-m;
+	transition: all tokens.$coral-transition-fast;
+	padding: tokens.$coral-size-xxs tokens.$coral-size-s;
+	max-width: 24rem;
+	text-align: start;
+
+	> span {
+		min-width: 0;
+	}
+
+	.buttonIcon {
+		height: tokens.$coral-size-m;
+		width: tokens.$coral-size-m;
+		margin-right: tokens.$coral-size-xxs;
+		flex-shrink: 0;
+		margin-bottom: $iconAlignmentOffsetM;
+	}
+
+	.buttonContent {
+		min-width: 0;
+	}
+
+	&:hover {
```

Do we change style on focus too?
```suggestion
	&:focus, &:hover {
```

---

Sure, let's